collections/Canon_Pat_Lat/MS_Canon_Pat_Lat_113.xml

```diff
@@ -41,46 +44,158 @@
-                     <decoNote>
-                        <p>Good initial. Scribbling sketch fol. 2r, 13th century (?). (P&amp;A i. 39)</p>
-                     </decoNote>
+                     <decoNote type="drawing">Sketch drawings of Christ Crucified, perhaps 13th-century (unfinished: one foot, and the Cross, are absent) (fol. 2r), head with halo (fol. 40r), and a wyvern(?) (fol. 42v); unfinished sketch, fol. 16r.</decoNote>
+                     <decoNote type="decInit">Vegetal initials, the first with red and silver (oxidized) (fols. 2v), some with a red ground (fols. 34r, 42r), others drawn in outline in red or brown ink (fols 4r, 22v, 23r, 34r, 60r, 61v).</decoNote>
+                     <decoNote type="colInit">Most initials in plain red, 1 to 3 lines high, sometimes with yellow wash.</decoNote>
                   </decoDesc>
+                  <bindingDesc>
+                     <binding notAfter="1810" notBefore="1700">
+                        <p>18th- or early 19th-century Italian binding, of rough black leather, the covered framed by triple gilt filets, the spine with a red leather title-piece lettered in gilt capitals ‘Cassiani | Collation | Libr | Co. mem. | perantiq’ (cf. MS. Canon. Misc. 575 et al.).</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1100" notBefore="1050">11th century, second half</origDate>
+                     <origDate calendar="Gregorian" notAfter="1200" notBefore="1050">11th century, second half (fol. 75: 12th century)</origDate>
                      <origPlace cert="low">
-                        <country key="place_7000084">German</country> (?)</origPlace>
+                        <country key="place_7000084">Germany</country> (?)</origPlace>
                   </origin>
-                  <provenance notAfter="1805" notBefore="1727" resp="#MMM"><persName key="person_2384880" role="fmo">Matteo Luigi Canonici, 1727–1805</persName></provenance>
+                  <provenance>The text and punctuation fairly extensively altered/corrected.</provenance>
+                  <provenance>Marginalia by at least one 15th-century hand (fol. 50r); a note about the misplaced bifolium by another (fol. 83v)</provenance>
+                  <provenance notAfter="1805" notBefore="1727" resp="#MMM"><persName key="person_2384880" role="fmo">Matteo Luigi Canonici, 1727–1805</persName>: source of acquisition unknown; not from Jacopo Soranzo.</provenance>
                   <provenance notAfter="1807" resp="#MMM"><persName key="person_446" role="fmo">Giuseppe Canonici , -1807</persName></provenance>
-                  <acquisition resp="#MMM" when="1817">Purchased by the Bodleian in 1817</acquisition>
+                  <acquisition resp="#MMM" when="1817">Purchased by the Bodleian in 1817. Former Bodleian shelfmarks: ‘Pat Lat. 113’ (fol. ir).</acquisition>
```

Comment:
Is that worth listing as a former shelfmark?

Reply:
probably not tbh, but Peter included it (as with similar cases where the old and current ones are very similar) and it doesn't do any harm to have it.